### ISSUE TITLE
Cropland: Malawi 2020

### ISSUE THREAD

**Issue by ivanzvonkov** [nasaharvest/crop-mask#219]
**Start year:** 2020
**Start month:** September

- [x] Labeling project created
- [x] Labeling completed
- [x] Data added to repository
- [x] Model trained (#294)
- [x] Map made

**Comment by ivanzvonkov** [comment]
Malawi dashboard: https://hkerner-umd.users.earthengine.app/view/malawi-cropland-in-2020-v1

**Comment by MsPixels** [comment]
Summary about the Malawi map quality from the call (09/27/2022)
- The terms 'crop type' and 'cropland' were clearly defined
- Incorporate shapefiles and other credible data sources that can improve the overall accuracy of the cropland maps

Next steps
- Use GEE for corrective data labeling for the cropland mapping 
-Christina and co to discuss the crop type/land use cases

**Comment by ivanzvonkov** [comment]
Related: https://github.com/nasaharvest/crop-mask/issues/222

**Comment by ivanzvonkov** [comment]
Got points from Blake will wait for #251 to finish

**Comment by MsPixels** [comment]
So far, I have received 28 corrective label files from Ivan. Tried adding the new labels but the Pull Request failed because of the unresolved billing issue

**Comment by MsPixels** [comment]
[Malawi Corrective Labelling Campaign](https://docs.google.com/document/d/1DfiWTmQKXqJLDJbS0_gmM409cebRu8xylxoUlyMjMjQ/edit)

**Comment by MsPixels** [comment]
Thanks Ivan. Yep, I shared this with Christina

**Comment by MsPixels** [comment]
@ivanzvonkov, this is the error I get when I run the model
![image (2)](https://user-images.githubusercontent.com/110104000/228039409-f830613a-c44a-4407-99ac-15e851075a18.png)

**Comment by ivanzvonkov** [comment]
On second thought, the test set metrics of the Malawi model trained on high quality data are higher than I've seen before (higher than validation set even), so maybe we should make a map:
https://github.com/nasaharvest/crop-mask/blob/c58a4426d5583d87eb48ab807c4a115437c9bb86/data/models.json#L70

**Comment by MsPixels** [comment]
After making the map, I got these results - [Comparing V1 and V2](https://code.earthengine.google.com/3bfaec4caa744f1b81e7ea0ee70622ab). There's definitely some changes

**Comment by ivanzvonkov** [comment]
Potentially do post processing (E.g. filtering out fallow fields re Ben's work, sieve for cleaning noisy predictions [window])

**Comment by MsPixels** [comment]
[Visualizing disagreement layer (V1 vrs V2](https://code.earthengine.google.com/fe099429cea286dc1612d10c300f4a0f)
